great_expectations/rule_based_profiler/data_assistant/visualization/plot_utils.py

```diff
@@ -0,0 +1,210 @@
+from typing import Dict, List, Optional
```

Comment:
Broad feedback: I'd like to move this plotting logic into the DataValidationResult; I'm not a fan of utils because they are hard to discover. I'd also like to remove DataValidator.plot(result) and instead use result.plot() exclusively.

Reply:
@donaldheppner @alexsherstinsky I am working on a PR that would do just that. I think we need to limit the scope of the PR as it is already massive and addresses a lot of other issues.

Reply:
@donaldheppner I assume by `DataValidator.plot()` you mean `DataAssistant.plot()`. I do think we have a need for `DataAssistant.get_metrics().plot()` though.